packages/lambdas/layers/analytics-platform/scripts/prep-deploy.sh

```
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+main() {
+   rm -rf ./dist/*
+   mkdir -p ./dist/configurations
+
+   cp -r ../../../data-transformation/fhir-to-csv/src/parseFhir/configurations/* ./dist/configurations/
+
+   pushd ./dist
```

Shellcheck (0.11.0)

[warning] Use 'pushd ... || exit' or 'pushd ... || return' in case pushd fails.

(SC2164)